src/main/java/net/sf/jabref/gui/groups/EntryTableTransferHandler.java

```diff
@@ -315,11 +315,9 @@ private boolean handleDraggedFilenames(String s, final int dropRow) {
      * @return success status for the operation
      */
     private boolean handleDraggedFiles(List<File> files, final int dropRow) {
-        final String[] fileNames = new String[files.size()];
-        int i = 0;
+        final List<String> fileNames = new ArrayList<>();
```

Comment:
I do not have any strong opinions but I know I made a PR a number of months
ago where I used these a lot and there were some objections.